{"name": "注册账号错误", "status": "passed", "parameters": [{"name": "case", "value": "{'data': 14, 'title': '注册账号错误'}"}], "start": 1614260767639, "stop": 1614260767639, "uuid": "cf3fae56-80ed-4a40-91d4-fd2a4644c2b0", "historyId": "d53648be26d53618b67d2ed4bdce73db", "testCaseId": "b89aea14a8418ebf1faf74844833ef92", "fullName": "testcases.test_login.TestRigers#test_login_error", "labels": [{"name": "story", "value": "登录功能"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestRigers"}, {"name": "host", "value": "MuSen"}, {"name": "thread", "value": "8296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login"}]}